firstlab.py

```diff
@@ -0,0 +1 @@
+print("Hello, world!")
```

Comment:
Hi! Welcome to the course!
All work concerning the first lab should be located in the lab_1_classify_by_unigrams folder.
Please, remove extra files.

Reply:
Please remove this file